Content.Server/GameObjects/Components/Interactable/WelderComponent.cs

```diff
@@ -116,7 +116,13 @@ private bool TryWeld(float value, IEntity? user = null, bool silent = false)
             if (_solutionComponent == null)
                 return false;
 
-            return _solutionComponent.TryRemoveReagent("chem.WeldingFuel", ReagentUnit.New(value));
+            bool succeeded = _solutionComponent.TryRemoveReagent("chem.WeldingFuel", ReagentUnit.New(value));
+
+            if (succeeded)
+            {
+                PlaySoundCollection("Welder");
```

Comment:
Maybe add a serialized data field for this instead of hardcoding it

Reply:
I actually had this serialized, but then i saw that the suicide method also uses the normal Welder soundcollection

Reply:
Make both serialized then :^)